---
title: ''
csl: msphere.csl
fontsize: 11pt
output:
  pdf_document:
    includes:
      in_header: header.tex
    keep_tex: yes
  word_document: default
  html_document:
    df_print: paged
geometry: margin=1.0in
bibliography: references.bib
---


```{r knitr_settings, eval=TRUE, echo=FALSE, cache=FALSE, warning=FALSE, message=FALSE, tidy=TRUE}
options(tidyverse.quiet = TRUE)

library(tidyverse)
library(knitr)

opts_chunk$set("tidy" = TRUE)
opts_chunk$set("echo" = FALSE)
opts_chunk$set("eval" = TRUE)
opts_chunk$set("warning" = FALSE)
opts_chunk$set("message" = FALSE)
opts_chunk$set("cache" = FALSE)

inline_hook <- function(x){
	print(x)

	if(is.list(x)){
		x <- unlist(x)
	}

	if(is.numeric(x)){
		if(abs(x - round(x)) < .Machine$double.eps^0.5){
			paste(format(x,big.mark=',', digits=0, scientific=FALSE))
		} else {
			paste(format(x,big.mark=',', digits=2, nsmall=2, scientific=FALSE))
		}
	} else {
    	paste(x)      
	}
}
knitr::knit_hooks$set(inline=inline_hook)
```



\vspace{35mm}

# The initial gut microbiota and response to antibiotic perturbation influence *Clostridioides difficile* clearance in mice

\vspace{35mm}


Sarah Tomkovich${^1}$, Joshua M.A. Stough${^1}$, Lucas Bishop${^1}$, Patrick D. Schloss^1$\dagger$^

\vspace{40mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI 48109




\newpage
\linenumbers


## Abstract
The gut microbiota has a key role in determining susceptibility to *Clostridioides difficile* infections (CDIs). However, much of the mechanistic work examining CDIs in mouse models use animals obtained from a single source. We treated mice from 6 sources (2 University of Michigan colonies and 4 commercial vendors) with clindamycin, followed by a *C. difficile* challenge and then measured *C. difficile* colonization levels throughout the infection. The microbiota were profiled via 16S rRNA gene sequencing to examine the variation across sources and alterations due to clindamycin treatment and *C. difficile* challenge. While all mice were colonized 1-day post-infection, variation emerged from days 3-7 post-infection with animals from some sources colonized with *C. difficile* for longer and at higher levels. We identified bacteria that varied in relative abundance across sources and throughout the experiment. Some bacteria were consistently impacted by clindamycin treatment in all sources of mice including *Lachnospiraceae*, *Ruminococcaceae*, and *Enterobacteriaceae*. To identify bacteria that were most important to colonization regardless of the source, we created logistic regression models that successfully classified mice based on whether they cleared *C. difficile* by 7 days post-infection using community composition data at baseline, post-clindamycin, and 1-day post-infection. With these models, we identified 4 bacteria that were predictive of whether *C. difficile* cleared. They varied across sources (*Bacteroides*), were altered by clindamycin (*Porphyromonadaceae*), or both (*Enterobacteriaceae* and *Enterococcus*). Allowing for microbiota variation across sources better emulates human inter-individual variation and can help identify bacterial drivers of phenotypic variation in the context of CDIs.

## Importance
*Clostridioides difficile* is a leading nosocomial infection. Although perturbation to the gut microbiota is an established risk, there is variation in who becomes asymptomatically colonized, develops an infection, or has adverse infection outcomes. Mouse models of *C. difficile* infection (CDI) are widely used to answer a variety of *C. difficile* pathogenesis questions. However, the inter-individual variation between mice from the same breeding facility is less than what is observed in humans. Therefore, we challenged mice from 6 different breeding colonies with *C. difficile*. We found that the starting microbial community structures and *C. difficile* persistence varied by the source of mice. Interestingly, a subset of the bacteria that varied across sources were associated with how long *C. difficile* was able to colonize. By increasing the inter-individual diversity of the starting communities, we were able to better model human diversity. This provided a more nuanced perspective of *C. difficile* pathogenesis.

\newpage

## Introduction

Antibiotics are a common risk factor for *Clostridioides difficile* infections (CDIs) due to their effect on the intestinal microbiota, but there is variation in who goes on to develop severe or recurrent CDIs after exposure [@Teng2019; @Kelly2012]. Additionally, asymptomatic colonization, where *C. difficile* is detectable, but symptoms are absent, has been documented in infants and adults [@Zacharioudakis2015; @Crobach2018]. The intestinal microbiota has been implicated in asymptomatic colonization [@Zhang2015; @VanInsberghe2020], susceptibility to CDIs [@Mancabelli2017; @Duvallet2017], and adverse CDI outcomes [@Seekatz2016; @Khanna2016; @Pakpour2017; @Lee2020]. However, it is not clear how much inter-individual microbiota variation contributes to the range of outcomes observed after *C. difficile* exposure relative to other risk factors.

Mouse models of CDIs have been a great tool for understanding *C. difficile* pathogenesis [@Hutton2014]. The number of CDI mouse model studies has grown substantially since Chen et al. published their C57BL/6 model in 2008, which disrupted the gut microbiota with antibiotics to enable *C. difficile* colonization and symptoms such as diarrhea and weight loss [@Chen2008]. CDI mouse models have been used to examine translationally relevant questions regarding *C. difficile*, including the role of the microbiota and efficacy of potential therapeutics for treating CDIs [@Best2012]. However, variation in the microbiota between mice from the same breeding colony is much less than the inter-individual variation observed between humans [@Baxter2014; @Nagpal2018]. Studying CDIs in mice with a homogeneous microbiota is likely to overstate the importance of individual mechanisms. Using mice that have a more heterogeneous microbiota would allow researchers to identify and validate more generalizable mechanisms responsible for CDI.

In the past, our group has attempted to introduce more variation into the mouse microbiota by using a variety of antibiotic treatments [@Reeves2011; @Schubert2015; @Jenior2017; @Jenior2018]. An alternative approach to maximize microbiota variation is to use mice from multiple sources [@Velazquez2019; @Osbelt2020]. The differences between the microbiota of mice from vendors have been well documented and shown to influence susceptibility to a variety of diseases [@Stough2016; @Alegre2019], including enteric infections [@EtienneMesmin2017; @Velazquez2019; @Osbelt2020; @Lai2020; @Thiemann2017; @Rolig2013; @Ge2018]. Different research groups have also observed different CDI outcomes despite using similar murine models [@Hutton2014; @Lawley2013; @Reeves2011; @Buffie2011; @Jenior2018; @Buffie2014]. Here we examined how variation in the baseline microbiota and responses to clindamycin treatment in C57BL/6 mice from six different sources influenced susceptibility to *C. difficile* colonization and the time needed to clear the infection.

## Results
```{r alpha}
richness_stats_all <- read_tsv("../data/process/richness_stats_all_days.tsv")
padj_dn1_richness <- richness_stats_all %>% filter(day == -1) %>% pull(p.value.adj)

shannon_stats_all <- read_tsv("../data/process/shannon_stats_all_days.tsv")
padj_dn1_shannon <- shannon_stats_all %>% filter(day == -1) %>% pull(p.value.adj)
```

```{r adonis}
adonis_stats_all <- read_tsv("../data/process/adonis_all.tsv")
r_sq_ordered_adonis <- adonis_stats_all %>% arrange(desc(r_sq))
rsq_adonis_source <- r_sq_ordered_adonis %>% pull(r_sq) %>% max
p_adonis_source <- r_sq_ordered_adonis %>% pull(p) %>% max

adonis_stats_dn101 <- read_tsv("../data/process/adonis_dn1-1.tsv")
dn1_rsq_sum <- adonis_stats_dn101 %>% filter(day == -1) %>% select(r_sq) %>% sum
d0_rsq_sum <-  adonis_stats_dn101 %>% filter(day == 0) %>% select(r_sq) %>% sum
d1_rsq_sum <-  adonis_stats_dn101 %>% filter(day == 1) %>% select(r_sq) %>% sum
p_adonis_all <- adonis_stats_dn101 %>% select(p) %>% max 
p_adonis_min <- adonis_stats_dn101 %>% select(p) %>% min #All p values are same

source_adonis_dn1 <- read_tsv("../data/process/adonis_dn1_source.tsv")
adonis_schloss_rsq_sum <- source_adonis_dn1 %>% filter(source == "Schloss") %>% select(r_sq) %>% sum
p_adonis_schloss_max <- source_adonis_dn1 %>% filter(source == "Schloss") %>% select(p) %>% max
adonis_young_rsq_sum <- source_adonis_dn1 %>% filter(source == "Young") %>% select(r_sq) %>% sum
p_adonis_young_max <- source_adonis_dn1 %>% filter(source == "Young") %>% select(p) %>% max
```

```{r source_taxa}
#Number of different otus across sources at day -1, 0, and 1 timepoints
source_otus <- read_tsv("../data/process/otu_stats_dn1to1_combined.tsv")
source_otus_dn1 <- source_otus %>% filter(day == -1) %>% nrow()
source_otus_d0 <- source_otus %>% filter(day == 0) %>% nrow()
source_otus_d1 <- source_otus %>% filter(day == 1) %>% nrow()

#Function to find which significant otus are shared 
intersect_all <- function(a,b,...){
  Reduce(intersect, list(a,b,...))
}

#OTUs shared across baseline, post-clindamycin, and post-infection timepoints
source_o_dn1 <- source_otus %>% filter(day == -1) %>% pull(otu)
source_o_d0 <- source_otus %>% filter(day == 0) %>% pull(otu)
source_o_d1 <- source_otus %>% filter(day == 1) %>% pull(otu)
shared_source_otus_dn1to1 <- length(intersect_all(source_o_dn1, source_o_d0, source_o_d1))

```

**The variation in the microbiota is high between mice from different sources.** We obtained C57BL/6 mice from 6 different sources: two colonies from the University of Michigan that were split from each other in 2010 (the Young and Schloss lab colonies) and four commercial vendors: the Jackson Laboratory, Charles River Laboratories, Taconic Biosciences, and Envigo (which was formerly Harlan). These 4 vendors were chosen because they are commonly used for murine CDI studies [@Spinler2016; @Markey2018; @McKee2018; @Yamaguchi2020; @Stroke2018; @EtienneMesmin2017; @Quigley2019; @Mullish2019]. Two experiments were conducted, approximately 3 months apart.  

We sequenced the V4 region of the 16S rRNA gene from fecal samples collected from these mice after they acclimated to the University of Michigan animal housing environment. We first examined the alpha diversity across the 6 sources of mice. There was a significant difference in the richness (i.e. number of observed operational taxonomic units (OTUs)), but not Shannon diversity index across the sources of mice (*P*~FDR~ = `r padj_dn1_richness` and *P*~FDR~ = `r padj_dn1_shannon`, respectively; Fig. 1A-B and Data Set S1, Sheets 1-2). Next, we compared the community structure of mice (Fig. 1C). The source of mice and the interactions between the source and cage effects explained most of the observed variation between fecal communities (PERMANOVA combined R^2^ = `r dn1_rsq_sum`, *P* `r format.pval(p_adonis_all, eps = 0.001)`; Fig. 1C and Data Set S1, Sheet 3). Mice that are co-housed tend to have similar gut microbiotas due to coprophagy [@Nguyen2015]. Since mice within the same source were housed together, it was not surprising that the cage effect also contributed to the observed community variation. There were some differences between the 2 experiments we conducted, as the experiment and cage effects significantly explained the observed community variation for the Schloss and Young lab mouse colonies (Fig. S1A-B and Data Set S1, Sheet 4). However, most of the vendors also clustered by experiment (Fig. S1C-D, F), suggesting there was some community variation between the 2 experiments within each source, particularly for Schloss, Young, and Envigo mice (Fig. S1G-H). After finding differences at the community level, we next identified the bacteria that varied between sources of mice. There were `r source_otus_dn1` OTUs with relative abundances that were significantly different between the sources at baseline (Fig. 1D and Data Set S1, Sheet 5). Though we saw differences between experiments at the community level, there were no OTUs that were significantly different between experiments within Schloss, Young, and Envigo mice at baseline (all *P* > 0.05). By using mice from six sources we were able to increase the variation in the starting communities to evaluate in a clindamycin-based CDI model.

```{r cfu}
cfu_stats_all <- read_tsv("../data/process/cfu_stats_all_days.tsv")
padj_d1_cfu <- cfu_stats_all %>% filter(day == 1) %>% pull(p.value.adj) 
median_d1 <- cfu_stats_all %>% filter(day == 1) %>% select(-day, -statistic, -p.value, -parameter, -method, -p.value.adj) %>% pivot_longer(everything(), values_to = "median_cfu", names_to = "Source") %>% arrange(median_cfu)
min_median_d1 <- median_d1 %>%  select(median_cfu) %>% min
max_median_d1 <- median_d1 %>% select(median_cfu) %>% max

cfu_stats_sig <- cfu_stats_all %>% filter(p.value.adj < 0.05)
max_padj_d3to7 <- cfu_stats_sig %>% select(p.value.adj) %>% max

#Percent of mice colonized with C. difficile each day
metadata <- read_tsv("../data/process/processed_metadata.tsv")
#Add clearance_status_dx columns for the rest of the days in addition to d7
metadata_cfu_status <- metadata %>% 
  mutate(clearance_status_d7 = case_when(cfu_d7 > 0 ~ "colonized",
                                         cfu_d7 == 0 ~ "cleared", #changed to match plot labeling used above
                                         cfu_d7 == NA ~ "NA"),
         clearance_status_d1 = case_when(cfu_d1 > 0 ~ "colonized",
                                         cfu_d1 == 0 ~ "cleared", 
                                         cfu_d1 == NA ~ "NA"),
         clearance_status_d2 = case_when(cfu_d2 > 0 ~ "colonized",
                                         cfu_d2 == 0 ~ "cleared", 
                                         cfu_d2 == NA ~ "NA"),
         clearance_status_d3 = case_when(cfu_d3 > 0 ~ "colonized",
                                         cfu_d3 == 0 ~ "cleared", 
                                         cfu_d3 == NA ~ "NA"),
         clearance_status_d4 = case_when(cfu_d4 > 0 ~ "colonized",
                                         cfu_d4 == 0 ~ "cleared", 
                                         cfu_d4 == NA ~ "NA"),
         clearance_status_d5 = case_when(cfu_d5 > 0 ~ "colonized",
                                         cfu_d5 == 0 ~ "cleared", 
                                         cfu_d5 == NA ~ "NA"),
         clearance_status_d6 = case_when(cfu_d6 > 0 ~ "colonized",
                                         cfu_d6 == 0 ~ "cleared", 
                                         cfu_d6 == NA ~ "NA"),
         clearance_status_d8 = case_when(cfu_d8 > 0 ~ "colonized",
                                         cfu_d8 == 0 ~ "cleared", 
                                         cfu_d8 == NA ~ "NA"),
         clearance_status_d9 = case_when(cfu_d9 > 0 ~ "colonized",
                                         cfu_d9 == 0 ~ "cleared", 
                                         cfu_d9 == NA ~ "NA"))


#Percent mice colonized each day, 2 experiments combined----
count_colonized <- function(timepoint, clearance_status_dx){
  metadata_cfu_status %>% 
    filter(day == timepoint, !is.na(cfu)) %>% #get rid of NA values
    count({{ clearance_status_dx }}) %>% 
    mutate(day = timepoint) %>% 
    rename(cfu_status = {{ clearance_status_dx }})
}  

#Figure out number of mice that either cleared C. difficile or were still colonized with C. difficile on each day
day_1 <- count_colonized(1, clearance_status_d1)
day_2 <- count_colonized(2, clearance_status_d2)
day_3 <- count_colonized(3, clearance_status_d3)
day_4 <- count_colonized(4, clearance_status_d4)
day_5 <- count_colonized(5, clearance_status_d5)
day_6 <- count_colonized(6, clearance_status_d6)
day_7 <- count_colonized(7, clearance_status_d7)
day_8 <- count_colonized(8, clearance_status_d8)
day_9 <- count_colonized(9, clearance_status_d9)

#Combine into a data frame and figure out percent colonized per day
percent_colonized_day <- rbind(day_1, day_2, day_3, day_4, day_5, day_6, day_7, day_8, day_9) %>% 
  group_by(day) %>% 
  mutate(total_n = sum(n)) %>% #Figure out total n per day (sum of cleared and colonized counts)
  filter(cfu_status == "colonized") %>% #Only need colonized rows to figure out percent colonized
  mutate(percent_colonized = (n/total_n) * 100)#Add new column for percent_colonized
percent_colonized_d7 <- percent_colonized_day %>% filter(day == 7) %>% 
  pull(percent_colonized)
n_day1 <- percent_colonized_day %>% filter(day == 1) %>% 
  pull(total_n)
n_day2 <- percent_colonized_day %>% filter(day == 2) %>% 
  pull(total_n)
n_day3 <- percent_colonized_day %>% filter(day == 3) %>% 
  pull(total_n)
n_day4 <- percent_colonized_day %>% filter(day == 4) %>% 
  pull(total_n)
n_day5 <- percent_colonized_day %>% filter(day == 5) %>% 
  pull(total_n)
n_day6 <- percent_colonized_day %>% filter(day == 6) %>% 
  pull(total_n)
n_day7 <- percent_colonized_day %>% filter(day == 7) %>% 
  pull(total_n)
n_day8 <- percent_colonized_day %>% filter(day == 8) %>% 
  pull(total_n)
n_day9 <- percent_colonized_day %>% filter(day == 9) %>% 
  pull(total_n)
```
**Clindamycin treatment renders all mice susceptible to *C. difficile* 630 colonization, but clearance time varies across sources.** Clindamycin is frequently implicated with human CDIs [@Guh2018] and was part of the antibiotic treatment for the frequently cited 2008 CDI mouse model [@Chen2008]. We have previously demonstrated mice are rendered susceptible to *C. difficile*, but clear the pathogen within 9 days, thus colonization is transient when treated with clindamycin alone [@Jenior2018; @Tomkovich2019]. All mice were treated with 10 mg/kg clindamycin via intraperitoneal injection and one day later challenged with 10^3^ *C. difficile* 630 spores (Fig. 2A). The day after infection, *C. difficile* was detectable in all mice at a similar level (median CFU range: `r format(min_median_d1, digits = 2, scientific=TRUE)`-`r format(max_median_d1, scientific=TRUE, digits = 2)`; *P*~FDR~ = `r padj_d1_cfu`), indicating clindamycin rendered all mice susceptible regardless of source (Fig. 2B). However, between 3 and 7 days post-infection, we observed variation in *C. difficile* levels across sources of mice (all *P*~FDR~ $\le$ `r max_padj_d3to7`; Fig. 2B and Data Set S1, Sheet 6). This suggested the source of mice was associated with *C. difficile* clearance. While the colonization dynamics were similar between the two experiments, the Schloss mice took longer to clear *C. difficile* in the first experiment compared to the second and the Envigo mice took longer to clear *C. difficile* in the second experiment compared to the first (Fig. S2A-B). The change in the mice's weight significantly varied across sources of mice with the most weight loss occurring two days post-infection (Fig. 2C and Data Set S1, Sheet 7). There was also one Jackson and one Envigo mouse that died between 1- and 3-days post-infection during the second experiment. Mice obtained from Jackson, Taconic, and Envigo tended to lose more weight, have higher *C. difficile* CFU levels and take longer to clear the infection compared to the other sources of mice (although there was variation between experiments with Schloss and Envigo mice). This was particularly evident 7 days post-infection (Fig. 2B-C, Fig. S2C-D), when `r format(percent_colonized_d7, digits = 1)`% of the mice were still colonized with *C. difficile* (Fig. S2E). By 9 days post-infection the majority of the mice from all sources had cleared *C. difficile* with the exception of 1 Taconic mouse from the first experiment and 2 Envigo mice from the second experiment  (Fig. 2B). Thus, clindamycin rendered all mice susceptible to *C. difficile* 630 colonization, regardless of source, but there was significant variation in disease phenotype across the sources of mice.

```{r clindamycin_taxa}
clindamycin_otus <- read_tsv("../data/process/otu_stats_dn1to0.tsv")
clindamycin_otus_sig <- clindamycin_otus %>% filter(p.value.adj < 0.05) %>% nrow()
```
**Clindamycin treatment alters bacteria in all sources, but a subset of bacterial differences across sources persists.** Given the variation in fecal communities that we observed across breeding colonies, we hypothesized that variation in *C. difficile* clearance would be explained by community variation across the 6 sources of mice. As expected, clindamycin treatment decreased the richness and Shannon diversity across all sources of mice (Fig. 3A-B). Interestingly, significant differences in diversity metrics between sources emerged after clindamycin treatment, with Charles River mice having higher richness and Shannon diversity than most of the other sources (*P*~FDR~ < 0.05; Fig 3A-B and Data Set S1, Sheets 1-2). The clindamycin treatment decreased the variation in community structures between sources of mice. The source of mice and the interactions between source and cage effects explained almost all of the observed variation between communities (combined R^2^ = `r d0_rsq_sum`, *P* `r format.pval(p_adonis_all, eps = 0.001)`; Fig. 3C and Data Set S1, Sheet 3). However, there were only `r source_otus_d0` OTUs with relative abundances that significantly varied between sources after clindamycin treatment (Fig. 3D and Data Set S1, Sheet 8). Next, we identified the bacteria that shifted after clindamycin treatment, regardless of source by analyzing paired fecal samples from mice that were collected at baseline and after clindamycin treatment. We identified `r clindamycin_otus_sig` OTUs that were altered after clindamycin treatment in most mice (Fig. 3E and Data Set S1, Sheet 9). When we compared the list of significant clindamycin impacted bacteria with the bacteria that varied between sources post-clindamycin, we found 4 OTUs that were shared between the lists (*Enterobacteriaceae* (OTU 1), *Lachnospiraceae* (OTU 130), *Lactobacillus* (OTU 6), *Enterococcus* (OTU 23); Fig. 3D-E and Data Set S1, Sheets 8-9).  Importantly, some of the OTUs that varied between sources also shifted with clindamycin treatment. For example, *Proteus* increased after clindamycin treatment (Fig. 3D), but only in Taconic mice. *Enterococcus* was primarily found in mice purchased from commercial vendors and also increased in relative abundance after clindamycin treatment (Fig. 3D). These findings demonstrate that clindamycin had a consistent impact on the fecal bacterial communities of mice from all sources and only a subset of the OTUs continued to vary between sources.

**Microbiota variation between sources is maintained after *C. difficile* challenge.** One day post-infection, significant differences in diversity metrics remained across sources (*P*~FDR~ < 0.05, Fig 4A-B and Data Set S1, Sheets 1-2). Although the Charles River mice had more diverse communities and were also able to clear *C. difficile* faster than the other sources, diversity did not explain the observed variation in *C. difficile* colonization across sources. The Young and Schloss mice had the lowest diversity 1 day post-infection and were able to clear *C. difficile* earlier than Jackson, Taconic and Envigo mice. The source of mice and the interactions between source and cage effects continued to explain most of the observed community variation (combined R^2^ = `r d1_rsq_sum`; *P* `r format.pval(p_adonis_all, eps = 0.001)`; Fig. 4C and Data Set S1, Sheet 3). One day after *C. difficile* challenge, there were `r source_otus_d1` OTUs with significantly different relative abundances across sources (Fig. 4D and Data Set S1, Sheet 10). 

Throughout the experiment, the source of mice continued to be the dominant factor that explained the observed variation across fecal communities (PERMANOVA R^2^ = `r rsq_adonis_source`, *P* `r format.pval(p_adonis_source, eps = 0.001)`) followed by interactions between cage effects  and the day of the experiment (Movie S1 and Data Set S1, Sheet 11). Fecal samples from the same source of mice continued to cluster closely to each other throughout the experiment. By 7 days post-infection, when approximately `r format(100-percent_colonized_d7, digits = 1)`% mice had cleared *C. difficile*, most of the mice had not recovered to their baseline community structure (Fig. 4E). The distance to the baseline community did not explain the variation in *C. difficile* clearance as the Schloss and Young mice had mostly cleared *C. difficile*, but their communities were a greater distance from baseline 7 days post-infection compared to the Jackson and Taconic mice that were still colonized. In summary, mouse bacterial communities varied significantly between sources throughout the course of the experiment and a consistent subset of bacteria remained different between sources regardless of clindamycin and *C. difficile* challenge.

```{r classification}
models_vs_random <- read_tsv("../data/process/classification_to_random.tsv")
max_p_vs_random <- models_vs_random %>% select(p.value) %>% max
median_otu_day0_AUROC <- models_vs_random %>% filter(model_name == "otu_day0") %>% pull(median)
median_otu_dayn1_AUROC <- models_vs_random %>% filter(model_name == "otu_dayn1") %>% pull(median)
median_otu_day1_AUROC <- models_vs_random %>% filter(model_name == "otu_day1") %>% pull(median)

padj_pairwise_comp <- read_tsv("../data/process/classification_model_pairwise_stats.tsv")
max_padj_otu_day0_comp <- padj_pairwise_comp %>% filter(str_detect(comparison, "otu_day0")) %>% select(p.value.adj) %>% max

#Function to find which significant otus are shared 
intersect_all <- function(a,b,...){
  Reduce(intersect, list(a,b,...))
}

#Read in components of venn diagram script to be able to calculate comparisons:----
#OTUs identified in logistic regression classification models (20 OTUs with the highest ranking for each model)
interp_otus <- read_tsv("../data/process/combined_top20_otus_all_models.tsv")
interp_otus_dn1 <- interp_otus  %>% filter(model_input_day == -1) %>% pull(OTU)
interp_otus_d0 <- interp_otus  %>% filter(model_input_day == 0) %>% pull(OTU)
interp_otus_d1 <- interp_otus  %>% filter(model_input_day == 1) %>% pull(OTU)
interp_combined <- c(interp_otus_dn1, interp_otus_d0, interp_otus_d1)

#List of OTUs that overlapped with day 0 models:
#Overlap between day 0 and day -1 model:
d0_vs_dn1 <- interp_otus %>% filter(model_input_day == 0 | model_input_day == -1) %>% filter(duplicated(OTU)) %>% pull(OTU)
#Overlap between day 0 and day 1 model:
d0_vs_d1 <- interp_otus %>% filter(model_input_day == 0 | model_input_day == 1) %>% filter(duplicated(OTU)) %>% pull(OTU)
#Overlap between day 0 model and the day -1 or 1 models:
overlapping_across_models <- c(d0_vs_dn1, d0_vs_d1)
overlapping_across_models <- str_replace(overlapping_across_models, " \\(.*\\)", "") #remove (OTU) from string
list_overlapping_across_models <- paste(overlapping_across_models, collapse= ", ") #Format to output in paper


#OTUs that vary across sources of mice on day -1, 0, or 1
sig_otu_source <- read_tsv("../data/process/otu_stats_dn1to1_combined.tsv")
`sig_otu_day-1` <- sig_otu_source %>% filter(day == -1) %>% pull(otu)
`sig_otu_day0` <- sig_otu_source %>% filter(day == 0) %>% pull(otu)
`sig_otu_day1` <- sig_otu_source %>% filter(day == 1) %>% pull(otu)
#OTUs that consistently vary across sources of mice on day-1, 0, or 1
shared_sig_otus_Dn1toD1 <- intersect_all(`sig_otu_day-1`, sig_otu_day0, sig_otu_day1)


#OTUs that were impacted by clindamycin treatment
`sig_otu_pairs` <- read_tsv("../data/process/otu_stats_dn1to0.tsv") %>% 
  filter(p.value.adj < 0.05) %>% pull(otu)

# Day -1 overlapping OTUs----
# Overlap between OTUs that varied across sources of mice on day -1 and top 20 taxa from logistic regression model based on day -1 community:
dayn1_and_interp_dn1_o <- intersect_all(`sig_otu_day-1`, `interp_otus_dn1`)
# Overlap between OTUs that were altered by clindamycin treatment and top 20 taxa from logistic regression model based on day -1 community:
paired_and_interp_dn1_o <- intersect_all(`sig_otu_pairs`, `interp_otus_dn1`)

# Day 0 overlapping OTUs----
# Overlap between OTUs that varied across sources of mice on day 0 and top 20 taxa from logistic regression model based on day 0 community:
day0_and_interp_d0_o <- intersect_all(`sig_otu_day0`, `interp_otus_d0`)
# Overlap between OTUs that were altered by clindamycin treatment and top 20 taxa from logistic regression model based on day 0 community:
paired_and_interp_d0_o <- intersect_all(`sig_otu_pairs`, `interp_otus_d0`)

# Day 1 overlapping OTUs----
# Overlap between OTUs that varied across sources of mice on day 1 and top 20 taxa from logistic regression model based on day 1 community:
day1_and_interp_d1_o <- intersect_all(`sig_otu_day1`, `interp_otus_d1`)
# Overlap between OTUs that were altered by clindamycin treatment and top 20 taxa from logistic regression model based on day 1 community:
paired_and_interp_d1_o <- intersect_all(`sig_otu_pairs`, `interp_otus_d1`)

#Combined overlapping OTUs based on day -1, 0, and 1 comparisons----
#Combined overlapping OTUs that varied by source and were in the top 20 taxa from at least one logistic regression model & remove any duplicates
dayn1to1_and_interp_combined <- unique(c(`dayn1_and_interp_dn1_o`, `day0_and_interp_d0_o`, `day1_and_interp_d1_o`))
#See which source OTUs show up as significant over multiple days
source_otus_w_duplicates <- c(`dayn1_and_interp_dn1_o`, `day0_and_interp_d0_o`, `day1_and_interp_d1_o`)
key_source_otus <- sort(source_otus_w_duplicates[duplicated(source_otus_w_duplicates)])
#Key source OTUs: "Bacteroides (OTU 2)", "Enterococcus (OTU 23)"
#Combined overlapping OTUs that were altered by clindamycin treatment and were in the top 20 taxa from at least one logistic regression model
paired_and_interp_combined <- unique(c(`paired_and_interp_dn1_o`, `paired_and_interp_d0_o`, `paired_and_interp_d1_o`))
#See which clindamycin-associated OTUs show up as significant over multiple days
clind_otus_w_duplicates <- c(`paired_and_interp_dn1_o`, `paired_and_interp_d0_o`, `paired_and_interp_d1_o`)
key_clind_otus <- sort(clind_otus_w_duplicates[duplicated(clind_otus_w_duplicates)])
#List of key OTUs from source and clindamycin varying OTUs:
key_otus <- unique(c(key_source_otus, key_clind_otus))
key_otus <- str_replace(key_otus, " \\(.*\\)", "") #remove (OTU) from string
list_key_otus <- paste(key_otus, collapse= ", ") #Format to output in paper

#Venn diagram calculations for OTUs----
venn_otus_source <- length(dayn1to1_and_interp_combined)
venn_otus_intersect <- length(intersect(dayn1to1_and_interp_combined, paired_and_interp_combined))
venn_otus_clind <-  length(paired_and_interp_combined)
```
**Baseline, post-clindamycin, and post-infection community data can predict mice that will clear *C. difficile* by 7 days post-infection.** After identifying taxa that varied between sources, changed after clindamycin treatment, or both, we determined which taxa were influencing the variation in *C. difficile* colonization at day 7 (Fig. 2B, Fig. S2C). We trained three L2-regularized logistic regression models with either input bacterial community data from the 6 sources of mice at the baseline (day = -1), post-clindamycin (day = 0), or post-infection (day = 1) timepoints of the experiment to predict *C. difficile* colonization status on day 7 (Fig. S3A-B). All models were better at predicting *C. difficile* colonization status on day 7 than random chance (all *P* `r format.pval(max_p_vs_random, eps = 0.001)`, Data Set S1, Sheet 12). The model based on the post-clindamycin (AUROC = `r median_otu_day0_AUROC`) community OTU data performed significantly better than the baseline (AUROC = `r median_otu_dayn1_AUROC`) or the post-infection (AUROC = `r median_otu_day1_AUROC`) models (*P*~FDR~ `r format.pval(max_padj_otu_day0_comp, eps = 0.001)` for pairwise comparisons; Fig. S3C and Data Set S1, Sheet 13). Thus, we were able to use bacterial relative abundance data from the time of *C. difficile* challenge to differentiate mice that had cleared *C. difficile* before day 7 from the mice still colonized with *C. difficile* at that timepoint. This result suggests that the bacterial community's response to clindamycin treatment had the greatest influence on subsequent *C. difficile* colonization dynamics.

To examine the bacteria that were driving each model’s performance, we selected the 20 OTUs that had the highest absolute feature weights in each of the 3 models (Data Set S1, Sheet 14). First, we looked at OTUs from the model with the best performance, which was based on the post-clindamycin treatment (day 0) bacterial community data. Out of the 10 highest ranked OTUs, 7 OTUs were associated with *C. difficile* colonization 7 days post-infection (*Bacteroides*, *Escherichia/Shigella*, 2 *Lachnospiraceae*,  *Lactobacillus*, *Porphyromonadaceae*, and *Ruminococcaceae*), while 3 OTUs were associated with clearance (*Enterobacteriaceae*, *Lachnospiraceae*, *Porphyromonadaceae*; Fig. 5A). On day 0, the majority of these OTUs were impacted by clindamycin and had relative abundances that were close to the limit of detection (Fig. 5A). Next, we examined whether any of the top 20 ranked OTUs from the post-clindamycin (day 0) model were also important in the other 2 classification models based on baseline (day -1) and 1 day post-infection community data. We identified 6 OTUs that were important to the post-clindamycin model and either the baseline or 1 day post-infection models (*`r list_overlapping_across_models`*; Data Set S1, Sheet 14). Thus, a subset of bacterial OTUs were important for determining *C. difficile* colonization dynamics across multiple timepoints.

To determine whether the OTUs driving the classification models also varied between sources, were altered by clindamycin treatment, or both, we identified the OTUs from each model that varied between sources (Fig. 1D, 3D, 4D and Data Set S1, Sheets 5, 8, and 10) or were impacted by clindamycin treatment (Fig. 3E and Data Set S1, Sheet 9; Fig. S4). Comparing the features important to the 3 models identified `r venn_otus_source` OTUs associated with source, `r venn_otus_clind` OTUs associated with clindamycin treatment, and `r venn_otus_intersect` OTUs associated with both (Fig. 5B). Together, these results suggest that the initial bacterial communities and their responses to clindamycin influenced the clearance of *C. difficile*.

Several OTUs that overlapped with our previous analyses appeared across at least 2 models (*`r list_key_otus`*), so we examined how the relative abundances of these OTUs varied over the course of the experiment (Fig. 6). Across the 9 days post-infection, there was at least 1 timepoint when the relative abundances of these OTUs significantly varied between sources (Data Set S1, Sheet 15). Interestingly, there were no OTUs that emerged as consistently enriched or depleted in mice that were colonized past 7 days post-infection, suggesting that multiple bacteria influence *C. difficile* colonization dynamics. 

## Discussion

Applying our CDI model to 6 different sources of mice, allowed us to identify bacterial taxa that were unique to different sources as well as taxa that were universally impacted by clindamycin. We trained L2-regularized logistic regression models with baseline (day -1), post-clindamycin treatment (day 0), and 1-day post-infection fecal community data that could predict whether mice cleared *C. difficile* by 7 days post-infection better than random chance. We identified *`r list_key_otus`* (Fig. 6) as candidate bacteria within these communities that influenced variation in *C. difficile* colonization dynamics since these bacteria were all important in the logistic regression models and varied by source, were impacted by clindamycin treatment, or both. Overall, our results demonstrated clindamycin was sufficient to render mice from multiple sources susceptible to CDI and only a subset of the inter-individual microbiota variation across mice from different sources was needed to predict which mice could clear *C. difficile*. 

Other studies have used mice from multiple sources to identify bacteria that either promote colonization resistance or increase susceptibility to enteric infections [@Velazquez2019; @EtienneMesmin2017; @Osbelt2020; @Lai2020; @Thiemann2017; @Rolig2013; @Ge2018]. For example, against *Salmonella* infections, *Enterobacteriaceae* and segmented filamentous bacteria have emerged as protective [@Velazquez2019; @Lai2020]. We found *Enterobacteriaceae* increased in all sources of mice after clindamycin treatment, positively correlating with *C. difficile* colonization. However, there was also variation in *Enterobacteriaceae* relative abundance levels between sources that was associated with the variation in *C. difficile* colonization dynamics across sources. Thus, bacteria may have differential roles in determining susceptibility depending on the type of bacterial infection.

Differences in CDI mouse model studies have been attributed to intestinal microbiota variation across sources. For example, researchers using the same clindamycin treatment and C57BL/6 mice had different *C. difficile* outcomes, one having sustained colonization [@Buffie2011], while the other had transient colonization [@Reeves2011], despite both using *C. difficile* VPI 10643. Baseline differences in the microbiota composition have been hypothesized to partially explain the differences in colonization outcomes and overall susceptibility to *C. difficile* after treatment with the same antibiotic [@Lawley2013; @Hutton2014]. When we treated mice from 6 different sources with clindamycin and challenged them with *C. difficile* 630, we found microbiota variation across sources impacted colonization outcomes, but not susceptibility. A previous study with *C. difficile* identified an endogenous protective *C. difficile* strain LEM1 that bloomed after antibiotic treatment in mice from Jackson or Charles River Laboratories, but not Taconic that protected mice against the more toxigenic *C. difficile* VPI10463 [@EtienneMesmin2017]. Given that we obtained mice from the same vendors, we checked all mice for endogenous *C. difficile* by plating stool samples that were collected after clindamycin treatment. However, we did not identify any endogenous *C. difficile* strains prior to challenge, suggesting there were no endogenous protective strains in the mice we received and other bacteria mediated the variation in *C. difficile* colonization across sources. The *C. difficile* strain used could also be contributing to the variation in *C. difficile* outcomes seen across different research groups. For example, a group found differential colonization outcomes after clindamycin treatment, with *C. difficile* 630 and M68 infections eventually becoming undetectable while strain BI-7 remained detectable up to 70 days post-treatment [@Lawley2012]. One study limitation is that we only used female mice. Sex has been shown to influence microbiota variation in mice [@Wang2019], so we used female mice to reduce this confounding variable and also match the sex used in previous CDI studies that administered clindamycin to mice [@Buffie2011; @Lawley2009; @Lawley2012; @Buffie2014]. The bacterial perturbations induced by clindamycin treatment have been well characterized and our findings agree with previous CDI mouse model work demonstrating *Enterococcus* and *Enterobacteriaceae* were associated with *C. difficile* susceptibility and *Porpyhromonadaceae*, *Lachnospiraceae*, *Ruminococcaceae*, and *Turicibacter* were associated with resistance [@Schubert2015; @Jenior2018; @Tomkovich2019; @Buffie2011; @Buffie2014; @Lawley2009; @Lawley2012; @Jump2014]. While we have demonstrated that susceptibility is uniform across sources of mice after clindamycin treatment, there could be different outcomes for either susceptibility or clearance in the case of other antibiotic treatments.  

We found the time needed to naturally clear *C. difficile* varied across sources of mice implying that at least in the context of the same perturbation, microbiota differences influence infection outcome. More importantly, we were able to explain the variation observed across sources with a subset of OTUs that were also important for predicting *C. difficile* colonization status 7 days post-infection. Since all but 3 mice eventually cleared *C. difficile* 630 by 9 days post-infection and the model built with the post-clindamycin (day 0) OTU relative abundance data had the best performance, our results suggest clindamycin treatment had a larger role in determining *C. difficile* susceptibility and clearance than the source of the mice.

Using mice from multiple sources successfully increased the inter-animal variation. One alternative approach that has been used in some CDI studies is to associate mice with human microbiotas [@NagaoKitamoto2020; @Battaglioli2018; @Robinson2014; @Collins2015; @Collins2018; @Hryckowian2018]. However, a major caveat to this method is the substantial loss of human microbiota community members upon transfer to mice [@Fouladi2020; @Walter2020]. Additionally, with the exception of 2 recent studies [@NagaoKitamoto2020; @Battaglioli2018], most of these studies associated mice with just 1 type of human microbiota either from a single donor or a single pool from multiple donors [@Robinson2014; @Collins2015; @Collins2018; @Hryckowian2018]. This approach does not aid in the goal of modeling the interpersonal variation seen in humans to understand how the microbiota influences susceptibility to CDIs and adverse outcomes. Importantly, our study using mice from 6 different sources increased the variation between groups of mice compared to using 1 source alone, to better reflect the inter-individual microbiota variation observed in humans. 

Another motivation for associating mice with human microbiotas is to study the bacteria associated with the disease in humans. Decreased *Bifidobacterium*, *Porphyromonas*, *Ruminococcaceae* and *Lachnospiraceae* and increased *Enterobacteriaceae*, *Enterococcus*, *Lactobacillus*, and *Proteus* have all been associated with human CDIs [@Mancabelli2017, @Duvallet2017]. Encouragingly, these populations were well represented in our study, suggesting most of the mouse sources are suitable for gaining insights into the bacteria influencing *C. difficile* colonization and infections in humans. An important exception was *Enterococcus*, which was primarily absent from University of Michigan colonies and *Proteus*, which was only found in Taconic mice. The fact that some CDI-associated bacteria were only found in a subset of mice has important implications for future CDI mouse model studies, but also models the natural patchiness of microbial populations in humans.

Other microbiota and host factors that were outside the scope of our current study may also contribute to the differences in *C. difficile* colonization dynamics between sources of mice. The microbiota is composed of viruses, fungi, and parasites in addition to bacteria, and these non-bacterial members can also vary across sources of mice [@Rasmussen2019; @Mims2020]. While our study focused solely on the bacterial portion, viruses and fungi have also begun to be implicated in the context of CDIs or fecal microbiota transplant (FMT) treatments for recurrent CDIs [@Markey2018; @Stewart2019; @Ott2017; @Zuo2017; @Zuo2018]. Beyond community composition, the metabolic function of the microbiota also has a CDI signature [@Robinson2019; @Jenior2017; @Fletcher2018; @Jump2014] and can vary across mice from different sources [@Xiao2015]. For example, microbial metabolites, particularly secondary bile acids and butyrate production, have been implicated as important contributors to *C. difficile* resistance [@Buffie2014; @Lawley2012]. Interestingly, butyrate has previously been shown to vary across mouse vendors and mediated resistance to *Citrobacter rodentium* infection, a model of enterohemorrhagic and enteropathogenic *Escherichia coli* infections [@Osbelt2020]. Evidence for immunological toning differences in IgA and Th17 cells across mice from different vendors have also been documented [@Fransen2015; @Ivanov2009] and could influence the host response to CDI [@Azrad2018; @Saleh2019], particularly relevant for *C. difficile* strains that induce more severe disease than *C. difficile* 630. The outcome after *C. difficile* exposure depends on a multitude of factors, including genetics, age, diet, and immunity; all of which also influence the microbiota. 

We have demonstrated that the ways baseline microbiotas from different mouse sources respond to clindamycin treatment influence the length of time mice remained colonized with *C. difficile* 630. To better understand the contribution of the microbiota to *C. difficile* pathogenesis and treatments, using multiple sources of mice may yield more insights than a single source. Furthermore, for studies wanting to examine the interplay between particular bacteria such as *Enterococcus* and *C. difficile*, these results could serve as a resource for selecting mice to address the question. Using mice from multiple sources helps model the interpersonal microbiota variation among humans to aid our understanding of how the gut microbiota provides colonization resistance to CDIs.

\newpage

## Acknowledgements
This work was supported by the National Institutes of Health (U01AI124255). ST was supported by the Michigan Institute for Clincial and Health Research Postdoctoral Translation Scholars Program (UL1TR002240 from the National Center for Advancing Translational Sciences). We thank members of the Schloss lab for feedback on planning the experiments and data presentation. In particular, we thank Begüm Topçuoğlu for help with implementing logistic regression models, Ana Taylor for help with media preparation and sample collection, and Nicholas Lesniak for his critical feedback on the manuscript. We also thank members of Vincent Young’s lab, particularly  Kimberly Vendrov, for guidance with the *C. difficile* infection mouse model and donating the mice. We also thank the Unit for Laboratory Animal Medicine at the University of Michigan for maintaining our mouse colony and providing the institutional support for our mouse experiments. Finally, we thank Kwi Kim, Austin Campbell, and Kimberly Vendrov for their help in maintaining the Schloss lab's anaerobic chamber.

\newpage

## Materials and Methods

**(i) Animals.** All experiments were approved by the University of Michigan Animal Care and Use Committee (IACUC) under protocol number PRO00006983. Female C57BL/6 mice were obtained from 6 different sources: The Jackson Laboratory, Charles River Laboratories, Taconic Biosciences, Envigo, and two colonies at the University of Michigan (the Schloss lab colony and the Young lab colony). The Young lab colony was originally established with mice purchased from Jackson in 2002, and the Schloss lab colony was established in 2010 with mice donated from the Young lab. The 4 groups of mice purchased from vendors were allowed to acclimate to the University of Michigan mouse facility for 13 days prior to starting the experiment. At least 4 female mice (age 5-10 weeks) were obtained per source and mice from the same source were primarily housed at a density of 2 mice per cage. The experiment was repeated once, approximately 3 months after the start of the first experiment.

**(ii) Antibiotic treatment.** After the 13-day acclimation period, all mice received 10 mg/kg clindamycin (filter sterilized through a 0.22 micron syringe filter prior to administration) via intraperitoneal injection (Fig. 1A).

**(iii) *C. difficile* infection model.** Mice were challenged with 10^3^ spores of *C. difficile* strain 630 via oral gavage post-infection 1 day after clindamycin treatment as described previously [@Jenior2018]. Mice weights and stool samples were taken daily through 9 days post-infection (Fig. 1A). Collected stool was split for *C. difficile* quantification and 16S rRNA sequencing analysis. For *C. difficile* quantification, stool samples were transferred to the anaerobic chamber, serially diluted in PBS, plated on taurocholate-cycloserine-cefoxitin-fructose agar (TCCFA) plates, and counted after 24 hours of incubation at 37&deg;C under anaerobic conditions. A sample from the day 0 timepoint (post-clindamycin and prior to *C. difficile* challenge) was also plated on TCCFA to ensure mice were not already colonized with *C. difficile* prior to infection. There were 3 deaths recorded over the course of the experiment, 1 Taconic mouse died prior to *C. difficile* challenge and 1 Jackson and 1 Envigo mouse died between 1- and 3-days post-infection. Mice were categorized as cleared when no *C. difficile* was detected in the first serial dilution (limit of detection: 100 CFU). Stool samples for 16S rRNA sequencing were snap frozen in liquid nitrogen and stored at -80&deg;C until DNA extraction.

**(iv) 16S rRNA sequencing.** DNA was extracted from -80&deg;C stored stool samples using the DNeasy Powersoil HTP 96 kit (Qiagen) and an EpMotion 5075 automated pipetting system (Eppendorf). The V4 region was amplified for 16S rRNA with the AccuPrime Pfx DNA polymerase (Thermo Fisher Scientific) using custom barcoded primers, as previously described [@Kozich2013]. The ZymoBIOMICS microbial community DNA standards was used as a mock community control [@Sze2019] and water was used as a negative control per 96-well extraction plate. The PCR amplicons were cleaned up and normalized with the SequalPrep normalization plate kit (Thermo Fisher Scientific). Amplicons were pooled and quantified with the KAPA library quantification kit (KAPA biosystems), prior to sequencing using the MiSeq system (Illumina).

**(v) 16S rRNA gene sequence analysis.** mothur (v. 1.43) was used to process all sequences [@Schloss2009] with a previously published protocol [@Kozich2013]. Reads were combined and aligned with the SILVA reference database [@Quast2012]. Chimeras were removed with the VSEARCH algorithm and taxonomic assignment was completed with a modified version (v16) of the Ribosomal Database Project reference database (v11.5) [@Cole2013] with an 80% confidence cutoff. Operational taxonomic units (OTUs) were assigned with a 97% similarity threshold using the opticlust algorithm [@Westcott2017]. Based on the mock communities, our overall sequencing error rate was 0.0112% and all  water controls had less than 1000 sequences (range: 18-875). To account for uneven sequencing across samples, samples were rarefied to 5,437 sequences 1,000 times for alpha and beta diversity analyses, and a single time to generate relative abundances for model training. PCoAs were generated based on the Yue and Clayton measure of dissimilarity ($\theta_{YC}$) distances [@Yue2005]. Permutational multivariate analysis of variance (PERMANOVA) was performed on mothur-generated $\theta_{YC}$ distance matrices with the adonis function in the vegan package [@Vegan2018] in R [@r_citation_2018].

**(vi) Classification model training and evaluation.** Models were generated based on mice that were categorized as either cleared or colonized 7 days post-infection and had sequencing data from the baseline (day -1), post-clindamycin (day 0), and post-infection (day 1) timepoints of the experiment. Input bacterial community relative abundance data at the OTU level from the baseline, post-clindamycin, and 1-day post-infection timepoints was used to generate 3 classification models that predicted *C. difficile* colonization status 7 days post-infection. The L2-regularized logistic regression models were trained and tested using the caret package [@Kuhn2008] in R as previously described [@Topcuoglu2020] with the exception that we used 60% training and 40% testing data splits for testing of the held out test data to measure model performance and repeated 2.5-fold cross-validation of the training data to select the best cost hyperparameter. The modified training to testing ratio was selected to accommodate the small number of samples in the dataset. Code was modified from https://github.com/SchlossLab/ML_pipeline_microbiome to update the classification outcomes and change the data split ratios. The modified repository to regenerate our modeling analysis is available at https://github.com/tomkoset/ML_pipeline_microbiome.

**(vii) Statistical analysis.** All statistical tests were performed in R (v `r paste(version$major, ".", version$minor, sep = "")`) [@r_citation_2018]. The Kruskal-Wallis test was used to analyze differences in *C. difficile* CFU, mouse weight change, and alpha diversity across sources with a Benjamini-Hochberg correction for testing multiple timepoints, followed by pairwise Wilcoxon comparisons with Benjamini-Hochberg correction. For taxonomic analysis and generation of logistic regression model input data, *C. difficile* (OTU 20) was removed. Bacterial relative abundances that varied across sources at the OTU level were identified with the Kruskal-Wallis test with Benjamini-Hochberg correction for testing all identified OTUs, followed by pairwise Wilcoxon comparisons with Benjamini-Hochberg correction. The Wilcoxon rank sum test was used to test for OTUs that differed between experiments within the Schloss, Young, and Envigo sources with Benjamini-Hochberg correction for testing all identified OTUs. OTUs impacted by clindamycin treatment were identified using the paired Wilcoxon signed rank test with matched pairs of mice samples from day -1 and day 0. To determine whether classification models had better performance (test AUROCs) than random chance (0.5), we used the one-sample Wilcoxon signed rank test. To examine whether there was an overall difference in predictive performance across the 3 classification models we used the Kruskal-Wallis test followed by pairwise Wilcoxan comparisons with Benjamini-Hochberg correction for multiple hypothesis testing. The tidyverse package (v `r paste(packageVersion("tidyverse")[1])`) was used to wrangle and graph data  [@Tidyverse2019].

**(viii) Code availability.** Code for all data analysis and generating this manuscript is available at https://github.com/SchlossLab/Tomkovich_Vendor_mSphere_2020.

**(ix) Data availability.** The 16S rRNA sequencing data have been deposited in the National Center for Biotechnology Information Sequence Read Archive (BioProject Accession no. PRJNA608529).

\newpage

## References

<div id="refs"></div>

\newpage

## Figures
```{r sample_size}
#figure out in functions.R and export results as a tsv to read in
analysis_n <- read_tsv("../data/process/processed_metadata.tsv")

#Initial N for each source from the 2 experiments combined:
schloss_i <- analysis_n %>% filter(day == -1 & vendor == "Schloss") %>% nrow()
young_i <- analysis_n %>% filter(day == -1 & vendor == "Young") %>% nrow()
jackson_i <- analysis_n %>% filter(day == -1 & vendor == "Jackson") %>% nrow()
cr_i <- analysis_n %>% filter(day == -1 & vendor == "Charles River") %>% nrow()
taconic_i <- analysis_n %>% filter(day == -1 & vendor == "Taconic") %>% nrow()
envigo_i <- analysis_n %>% filter(day == -1 & vendor == "Envigo") %>% nrow()

#Range in N for C. difficile CFU quantification:
cfu_n_by_day <- analysis_n %>% 
  filter(!is.na(cfu)) %>% #remove rows with NAs
  group_by(day) %>% count() %>% arrange(n) %>% ungroup()
min_cfu_n <- cfu_n_by_day %>% select(n) %>% min
max_cfu_n <- cfu_n_by_day %>% select(n) %>% max

#Range in N for mouse weight change data over the course of the experiment:
weight_n_by_day <- analysis_n %>% 
  filter(!is.na(weight)) %>% #remove rows with NAs
  group_by(day) %>% count() %>% arrange(n) %>% ungroup()
min_weight_n <- weight_n_by_day %>% select(n) %>% min
max_weight_n <- weight_n_by_day %>% select(n) %>% max

#C. difficile day 7 colonization status N range across sources:
day7_status_n <- analysis_n %>% 
  filter(!is.na(clearance_status_d7)) %>% #remove rows with NAs
  filter(day == 7) %>% 
  group_by(vendor) %>% count() %>% arrange(n) %>% ungroup
day7_status_n_min <- day7_status_n %>% select(n) %>% min
day7_status_n_max <-day7_status_n %>% select(n) %>% max

#N for mice with sequencing data on both day -1 and day 0 (used to determine taxa that were altered by clindamycin treatment)
n_paired_for_clind <- read_tsv("../data/process/vendors.diversity.summary") %>% #read in sequencing data & join to analysis_n
  filter(method == "ave") %>% 
  select(group, sobs, shannon, invsimpson, coverage) %>% 
  inner_join(analysis_n, by = c("group" = "id")) %>% 
  filter(day == -1 | day == 0) %>% 
  filter(duplicated(mouse_id)) %>% 
  nrow()
```
**Figure 1. Microbiota variation is high between mice from different sources.** A-B. Number of observed OTUs (A) and Shannon diversity index values (B) across sources of mice at baseline (day -1 of the experiment). Differences between sources were analyzed by Kruskal-Wallis test with Benjamini-Hochberg correction for testing each day of the experiment and the adjusted *P* value was < 0.05 for panel A (Data Set S1, Sheet 1). None of the *P* values from pairwise Wilcoxon comparisons between sources were significant after Benjamini-Hochberg correction (Data Set S1, Sheet 2). Gray lines represent the median values for each source of mice. C. Principal Coordinates Analysis (PCoA) of $\theta_{YC}$ distances of baseline stool samples. Source and the interaction between source and cage effects explained most of the variation (PERMANOVA combined R^2^ = `r dn1_rsq_sum`, *P* `r format.pval(p_adonis_all, eps = 0.001)`; Data Set S1, Sheet 3). For A-C:  each symbol represents the value for a stool sample from an individual mouse, circles represent experiment 1 mice and triangles represent experiment 2 mice. D. The median (point) and interquantile range (colored lines) of the relative abundances for the 20 most significant OTUs out of the `r source_otus_dn1` OTUs that varied across sources at baseline by Kruskal-Wallis test with Benjamini-Hochberg correction (Data Set S1, Sheet 5).

\newpage
**Figure 2. Clindamycin is sufficient to promote *C. difficile* colonization in all mice, but clearance time varies across sources.** A. Setup of the experimental timeline. Mice for the experiments were obtained from 6 different sources: the Schloss (N = `r schloss_i`) and Young lab (N = `r young_i`) colonies at the University of Michigan, the Jackson Laboratory (N = `r jackson_i`), Charles River Laboratory (N = `r cr_i`), Taconic Biosciences (N = `r taconic_i`), and Envigo (N = `r envigo_i`). Mice that were ordered from commercial vendors acclimated to the University of Michigan mouse facility for 13 days prior to antibiotic administration. All mice were administered 10 mg/kg clindamycin intraperitoneally (IP) 1 day before challenge with *C. difficile* 630 spores on day 0. Mice were weighed and feces was collected daily through the end of the experiment (9 days post-infection). Note: 3 mice died during course of experiment. 1 Taconic mouse prior to infection and 1 Jackson and 1 Envigo mouse between 1- and 3-days post-infection. B. *C. difficile* CFU/gram stool measured over time (N = `r min_cfu_n`-`r max_cfu_n` mice per timepoint) via serial dilutions. The black line represents the limit of detection for the first serial dilution. CFU quantification data was not available for each mouse due to early deaths, stool sampling difficulties, and not plating all of the serial dilutions. C. Mouse weight change measured in grams over time (N = `r min_weight_n`-`r max_weight_n` mice per timepoint), all mice were normalized to the weight recorded 1 day before infection. For B-C: timepoints where differences between sources of mice were statistically significant by Kruskal-Wallis test with Benjamini-Hochberg correction for testing across multiple days (Data Set S1, Sheets 6-7) are reflected by the asterisk above each timepoint (\*\, *P* < 0.05). Lines represent the median for each source and circles represent individual mice from experiment 1 while triangles represent mice from experiment 2. 

\newpage
**Figure 3. Clindamycin treatment alters bacteria in all sources, but a subset of bacterial differences across sources persists.** A-B. Number of observed OTUs (A) and Shannon diversity index values (B) across sources of mice after clindamycin treatment (day 0). Differences between sources were analyzed by Kruskal-Wallis test with Benjamini-Hochberg correction for testing each day of the experiment and the adjusted *P* value was < 0.05 (Data Set S1, Sheet 1). Significant *P* values from the pairwise Wilcoxon comparisons between sources with Benjamini-Hochberg correction are displayed as the first initial of each group compared to the group that they are listed above (Data Set S1, Sheet 2). C. PCoA of $\theta_{YC}$ distances from stools collected post-clindamycin. Source and the interaction between source and cage effects explained most of the variation observed post-clindamycin (PERMANOVA combined R^2^ = `r d0_rsq_sum`, *P* `r format.pval(p_adonis_all, eps = 0.001)`; Data Set S1, Sheet 3). For A-C, each symbol represents a stool sample from an individual mouse, with circles representing experiment 1 mice and triangles representing experiment 2 mice. D. The median (point) and interquantile range (colored lines) of the relative abundances for the `r source_otus_d0` OTUs (Data Set S1, Sheet 8) that varied between sources after clindamycin treatment (day 0). E. The median (point) and interquantile range (colored lines) of the top 10 most significant OTUs out of `r clindamycin_otus_sig` with relative abundances that changed because of the clindamycin treatment (adjusted *P* value < 0.05). Data were analyzed by paired Wilcoxon signed rank test of mice that had paired sequence data for baseline (day -1) and post-clindamycin (day 0) timepoints (N = `r n_paired_for_clind`), with Benjamini-Hochberg correction for testing all identified OTUs (Data Set S1, Sheet 9). The gray vertical line indicates the limit of detection. 

\newpage
**Figure 4. Microbiota variation across sources is maintained after *C. difficile* challenge.** A-B. Number of observed OTUs (A) and Shannon diversity index values (B) across sources of mice 1-day post-infection. Data were analyzed by Kruskal-Wallis test with Benjamini-Hochberg correction for testing each day of the experiment and the adjusted *P* value was < 0.05 (Data Set S1, Sheet 1). Significant *P* values from the pairwise Wilcoxon comparisons between sources with Benjamini-Hochberg correction are displayed as the first initial of each group compared to the group that they are listed above (Data Set S1, Sheet 2). PCoA of $\theta_{YC}$ distances of 1-day post-infection stool samples. Source and the interaction between source and cage effects explained most of the variation between fecal communities (PERMANOVA combined R^2^ = `r d1_rsq_sum`, *P* `r format.pval(p_adonis_all, eps = 0.001)`; Data Set S1, Sheet 3). For A-C:  each symbol represents the value for a stool sample from an individual mouse, circles represent experiment 1 mice and triangles represent experiment 2 mice. D. The median (point) and interquantile range (colored lines) of the relative abundances for the top 20 most significant OTUs out of the `r source_otus_d1` OTUs that varied between sources 1-day post-infection. The gray vertical line indicates the limit of detection. For each timepoint OTUs with differential relative abundances across sources of mice were identified by Kruskal-Wallis test with Benjamini-Hochberg correction for testing all identified OTUs (Data Set S1, Sheet 10). E. $\theta_{YC}$ distances of fecal samples collected 7-days post-infection relative to the baseline (day -1) sample for each mouse. Each symbol represents an individual mouse. Gray lines represent the median for each source.

\newpage
**Figure 5. Bacteria that influenced whether mice cleared *C. difficile* by day 7.** A. Post-clindamycin (day 0) relative abundance data for the 10 OTUs with the highest rankings based on feature weights in the post-clindamycin (day 0) classification model. Red font represents OTUs that correlated with *C. difficile* colonization and blue font represents OTUs that correlated with clearance. Symbols represent the relative abundance data for an individual mouse. Gray bars indicate the median relative abundances for each source. The gray horizontal lines indicate the limit of detection. B. Venn diagram that combines  OTUs that were important to the day -1, 0, and 1 classification models ( Fig. S4, Data Set S1, Sheet 14) and either overlapped with taxa that varied across sources at the same timepoint, were impacted by clindamycin treatment, or both. Bold OTUs were important to more than 1 classification model.

\newpage
**Figure 6: OTUs associated with *C. difficile* colonization dynamics vary across sources throughout the experiment.** A-D. Relative abundances of bold OTUs from Fig. 5B that were important in at least two classification models are shown over time. A. *Bacteroides* (OTU 2), which varied across sources throughout the experiment. B-C. *Enterobacteriaceae* (B) and *Enterococcus* (C), which significantly varied across sources and were impacted by clindamycin treatment. D. *Porphyromonadaceae* (OTU 7), which was significantly impacted by clindamycin treatment and after examining relative abundance dynamics over the course of the experiment was found to also significantly vary between sources of mice on days -1, 5, 6, 7, and 9 of the experiment. Symbols represent the relative abundance data for an individual mouse. Colored lines indicate the median relative abundances for each source. The gray horizontal line represents the limit of detection. Timepoints where differences between sources of mice were statistically significant by Kruskal-Wallis test with Benjamini-Hochberg correction for testing across multiple days (Data Set S1, Sheet 15) are identified by the asterisk above each timepoint (\*\, P < 0.05).

\newpage
**Figure S1. Bacterial communities vary between experiments for some sources.** A-F. PCoA of $\theta_{YC}$ distances for the baseline fecal bacterial communities within each source of mice. Each symbol represents a stool sample from an individual mouse with color corresponding to experiment and shape representing cage mates. Experiment number and cage effects explained most of the observed variation for samples from the Schloss (PERMANOVA combined R^2^ = `r adonis_schloss_rsq_sum`; *P* $\le$ `r p_adonis_schloss_max`) and Young (combined R^2^ = `r adonis_young_rsq_sum`; *P* $\le$ `r p_adonis_young_max`) mice (Data Set S1, Sheet 4). G-H: Boxplots of the $\theta_{YC}$ distances of the 6 sources of mice relative to mice within the same source and experiment (G) or mice within the same source and between experiments (H) at baseline (day -1). Symbols represent individual mouse samples: circles for experiment 1 and triangles for experiment 2. 

\newpage
**Figure S2. *C. difficile* CFU variation across sources varies slightly between the 2 experiments.** A-B. *C. difficile* CFU/gram of stool quantification over time for experiment 1 (A) and 2 (B). Experiments were conducted approximately 3 months apart. Lines represent the median CFU for each source, symbols represent individual mice and the black line represents the limit of detection. C. *C. difficile* CFU/gram stool 7-days post-infection across sources of mice with an asterisk for pairwise Wilcoxon comparisons with Benjamini-Hochberg correction where *P* < 0.05. D. Mouse weight change 2-days post-infection across sources of mice, no pairwise Wilcoxon comparisons were significant after Benjamini-Hochberg correction. For C-D: circles represent experiment 1 mice, triangles represent experiment 2 mice and gray lines indicate the median values for each group. E. Percent of mice that were colonized with *C. difficile* over the course of the experiment. Each day the percent is calculated based on the mice where *C. difficile* CFU was quantified for that particular day. Total N for each day: day 1 (N = `r n_day1`), day 2 (N = `r n_day2`), day 3 (N = `r n_day3`), day 4 (N = `r n_day4`), day 5 (N = `r n_day5`), day 6 (N = `r n_day6`), day 7 (N = `r n_day7`), day 8 (N = `r n_day8`), and day 9 (N = `r n_day9`).

\newpage
**Figure S3. Bacterial community composition before, after clindamycin perturbation, and post-infection can predict *C. difficile* colonization status 7 days post-infection.** A. Bar graph visualizations of overall 7-days post-infection *C. difficile* colonization status that were used as classification outcomes to build L2-regularized logistic regression models. Mice were classified as colonized or cleared (not detectable at the limit of detection of 100 CFU) based on CFU g/stool data from 7 days post-infection. B. *C. difficile* CFU status on Day 7 within each mouse source. N = `r day7_status_n_min`-`r day7_status_n_max` mice per group. C. L2-regularized logistic regression classification model area under the receiving operator characteristic curve (AUROCs) to predict *C. difficile* CFU on day 7 post-infection (Fig. 2B, Fig. S2C) based on the OTU community relative abundances at baseline (day -1), post-clindamycin (day 0), and 1-day post-infection. All models performed better than random chance (AUROC = 0.5, all *P* `r format.pval(max_p_vs_random, eps = 0.001)`, Data Set S1, Sheet 12) and the model built with post-clindamycin bacterial OTU relative abundances had the best performance ((*P*~FDR~ `r format.pval(max_padj_otu_day0_comp, eps = 0.001)` for all pairwise comparisons, Data Set S1, Sheet 13). See Data Set S1, Sheet 14 for list of the 20 OTUs that were ranked as most important to each model.

\newpage
**Figure S4. OTUs from classification models based on baseline, post-clindamycin treatment, or 1-day post-infection community data vary by source, clindamycin treatment, or both.** A-C. Venn diagrams of OTUs from the top 20 OTUs from the baseline (A), post-clindamycin treatment (B), and 1-day post-infection (C) classification models (Data Set S1, Sheet 14) that overlapped with OTUs that varied across sources at the corresponding timepoint (Data Set S1, Sheets 5, 8, and 10), were impacted by clindamycin treatment (Data Set S1, Sheet 9), or both. Bold OTUs were important to more than 1 classification model.

\newpage
## Supplementary Movie and Data Set S1

**Movie S1. Large shifts in bacterial community structures occurred after clindamycin and *C. difficile* infection.** PCoA of $\theta_{YC}$ distances animated from days -1 through 9 of the experiment. Source was the variable that explained the most observed variation across fecal communities (PERMANOVA source R^2^ = `r rsq_adonis_source`, *P* = `r p_adonis_source`, Data Set S1, Sheet 11) followed by interactions between cage effects and day of the experiment. Transparency of the symbol corresponds to the day of the experiment, each symbol represents a sample from an individual mouse at a specific timepoint. Circles represent mice from experiment 1 and triangles represent mice from expeirment 2.

**Data Set S1, Sheets 1-15. Excel workbook with 15 sheets.**
  
**Data Set S1, Sheet 1. Alpha diversity metrics Kruskal-Wallis statistical results.**
  
**Data Set S1, Sheet 2. Alpha diversity metrics pairwise Wilcoxon statistical results.**
  
**Data Set S1, Sheet 3. PERMANOVA results for mice at baseline (day -1), post-clindamycin (day 0), and post-infection (day 1).**
  
**Data Set S1, Sheet 4. PERMANOVA results for each source of mice at baseline (day -1).**
  
**Data Set S1, Sheet 5. OTUs with relative abumdances that significantly vary between sources at baseline (day -1).**
  
**Data Set S1, Sheet 6. *C. difficile* CFU statistical results.**
  
**Data Set S1, Sheet 7. Mouse weight change statistical results.**
  
**Data Set S1, Sheet 8. OTUs with relative abundances that significantly vary between sources post-clindamycin (day 0).**
  
**Data Set S1, Sheet 9. OTUs with relative abundances that significantly changed after clindamycin treatment.**
  
**Data Set S1, Sheet 10. OTUs with relative abundances that significantly vary between sources 1-day post-infection.**
  
**Data Set S1, Sheet 11. PERMANOVA results for mice across all timepoints.**
  
**Data Set S1, Sheet 12. Statistical results of L2-regularized logistic regression model performances compared to random chance.**
  
**Data Set S1, Sheet 13. Pairwise comparisons of L2-regularized logistic regression model performances.**
  
**Data Set S1, Sheet 14. Top 20 most important OTUs for each of the 3 L2-regularized logistic regression models based on OTU relative abundance data.**
  
**Data Set S1, Sheet 15. OTUs with relative abundances that significantly varied between sources of mice on at least 1 day of the experiment by Kruskal-Wallis test.**



